exercises/004_palindrome.js

```diff
@@ -0,0 +1,18 @@
+//4. Enter a  string and find if it's a palindrome
+const readlineSync = require("readline-sync");
+const string = readlineSync.question("Enter a String:")
+
+const palindrome = (string) => {
+    let len = string.length;
+    str = string.toLowerCase();
```

Comment:
str is not declared anywhere. use let or const. Else it will get a global scope

Reply:
made the changes